mlir/test/Conversion/AIRLowering/air_channel_get_put.mlir

```diff
@@ -24,25 +24,26 @@ module {
       %c32 = arith.constant 32 : index
       %c1 = arith.constant 1 : index
       %c0 = arith.constant 0 : index
-      %0 = air.channel.put async  @channel_0[%c0, %c0] (%arg0[%c8, %c0] [%c8, %c16] [%c32, %c0]) {id = 1 : i32} : (memref<32x16xi32>)
-      %1 = air.channel.get async  @channel_1[%c0, %c0] (%arg1[%c8, %c0] [%c8, %c16] [%c32, %c0]) {id = 2 : i32} : (memref<32x16xi32>)
+      %0 = air.channel.put async  @channel_0[%c0, %c0] (%arg0[%c8, %c0] [%c8, %c16] [%c32, %c1]) {id = 1 : i32} : (memref<32x16xi32>)
+      %1 = air.channel.get async  @channel_1[%c0, %c0] (%arg1[%c8, %c0] [%c8, %c16] [%c32, %c1]) {id = 2 : i32} : (memref<32x16xi32>)
```

Comment:
Are you changing the semantics of get and put in this PR? Please give a longer description of what you are changing.

Reply:
Sure. Some of the original tests, where the last stride value was set to 0, was using inconsistent semantics compared with the rest of the tests. The original intent of these tests should be "accessing data in row-major, consecutive" order in the inner most stride dimension. That means the last stride value should be 1.

Those tests could get away with that inconsistency because the last stride dimension is simply thrown away, as the downstream aiex.npu.dma op only has stride dimensions from 1 to 3, where the lowest dimension is assumed to be always 1.

Reply:
I see, thanks. The lowering always ignores the inner stride so there was never any issue.